dotnet/src/Skills/Skills.MsGraph/TaskListSkill.cs

```diff
@@ -90,7 +90,7 @@ public static DateTimeOffset GetNextDayOfWeek(DayOfWeek dayOfWeek, TimeSpan time
             title: title,
             reminder: reminder);
 
-        this._logger.LogInformation("Adding task '{0}' to task list '{1}'", task.Title, defaultTaskList.Name);
+        this._logger.LogTrace("Adding task '{0}' to task list '{1}'", task.Title, defaultTaskList.Name);
```

Comment:
Is this sensitive?

Reply:
I think that task title might be sensitive, as it may contain PII data. Happy to chat about different cases.

Reply:
Nah, all good. I was just wondering whether to recommend the // Sensitive data comment here too